### PR TITLE
feat: regenerated

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@multi-geth/rpc-client",
+  "name": "template-client",
```
Comment (BelfordZ) [inline package.json]:
```suggestion
  "name": "@multi-geth/rpc-client",
```

File: package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   "version": "1.0.0",
```
Comment (BelfordZ) [inline package.json]:
```suggestion
  "version": "1.0.1",
```

File: docs/.nojekyll [hunk 3]
```diff
@@ -1 +0,0 @@
-
```
Comment (shanejonas) [inline docs/.nojekyll:1]: we need this

**Review by shanejonas — CHANGES_REQUESTED** [review]
dont delete `docs/.nojekyll`

File: docs/.nojekyll (hunk 3 above)
Comment (BelfordZ) [reply docs/.nojekyll:1]: non scripted client - 100
us - 0

**Pull request closed without merging**